### ISSUE TITLE
input validation in calc_FastRatio()

### ISSUE THREAD

**Issue by mcol** [R-Lum/Luminescence#157]
There are a number of checks that are currently missing from `calc_FastRatio()`:

* `Ch_L2` should be a positive integer
* `Ch_L3` should have start < end (@RLumSK is start <= end fine too?)
* elements in `Ch_L3` should not exceed the number of rows in the input object and should be positive integers

At least following are tests that should be added:
```R
calc_FastRatio(ExampleData.CW_OSL_Curve, Ch_L2 = -1)
calc_FastRatio(ExampleData.CW_OSL_Curve, Ch_L2 = 0)
calc_FastRatio(ExampleData.CW_OSL_Curve, Ch_L2 = c(1, 2))
calc_FastRatio(ExampleData.CW_OSL_Curve, Ch_L3 = c(5, 2))
calc_FastRatio(ExampleData.CW_OSL_Curve, Ch_L3 = c(5, 1001))
calc_FastRatio(ExampleData.CW_OSL_Curve, Ch_L3 = c(1001, 2001))
```

**Comment by RLumSK** [comment]
> * `Ch_L3` should have start < end (@RLumSK is start <= end fine too?)

Yes, thanks!

**Comment by mcol** [comment]
What should happen when a channel value exceeds the number of rows in the input object? Should we throw an error, or ignore the user setting and throw a warning?

**Comment by mcol** [comment]
Fixed by #159.
